tools/dds/dds-server/rs-dds-server.cpp

```diff
@@ -90,16 +96,122 @@ try
         return EXIT_FAILURE;
     }
 
+    std::map<rs2::device, std::pair<std::shared_ptr<librealsense::dds::dds_device_server>, std::shared_ptr<tools::lrs_device_manager>>> device_handlers_list;
     
     std::cout << "Start listening to RS devices.." << std::endl;
+
+    // Create a RealSense context
+    rs2::context ctx( "{"
+        "\"dds-discovery\" : false"
+        "}" );
+
     // Run the LRS device watcher
-    tools::lrs_device_watcher dev_watcher;
-    dev_watcher.run( [&]( rs2::device dev ) { broadcaster.add_device( dev ); },
-                     [&]( rs2::device dev ) { broadcaster.remove_device( dev ); } );
+    tools::lrs_device_watcher dev_watcher( ctx );
+    dev_watcher.run(
+        // Handle a device connection
+        [&]( rs2::device dev ) {
```

Comment:
This is a very big lambda. Consider breaking into a separate function.

Reply:
I agree, will do

Reply:
Done